03-Custom-Actions-In-Python/06-Improving-Performance.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>I think in the first line you can remove the comma after considerations.</p><p> </p><p>I'm not quite sure what you mean by "However, first profile and then profile".</p><p> </p><p>Is there a tutorial that is going to be specifically about local snapshots? If not, I think some more explanation about them is warranted in this section.</p><p> </p><p>Rephrase "If accessing state information using local snapshots (...), will typically be faster. They don't copy data or aggregate across MPI ranks." to "Local snapshots (...) provide faster access to the simulation's state information than traditional snapshots because they don't copy data or aggregate across MPI ranks".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>